docs/examples/frameworks/jax/pax-basic_example.ipynb

```diff
@@ -0,0 +1,1482 @@
+{
```

Comment:
<p>Can you maybe describe, that this MnistDaliInput replaces MnistDatasetInput in the original example which can be found in "docs/examples/frameworks/jax/pax_examples/dali_pax_example.py"? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I reworded this whole section and added more information below the code. I added link to the code that should work when we merge it. Alternatively we can skip the link for now and I can add it in the next PR.